# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gst-plugins/dvdread-compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gst-plugins-0.8.11/ext/dvdread/dvdreadsrc.c.orig	2006-02-06 13:11:34.000000000 +0100
+++ gst-plugins-0.8.11/ext/dvdread/dvdreadsrc.c	2006-02-06 13:11:53.000000000 +0100
@@ -37,6 +37,7 @@
 //#include <linux/cdrom.h>
 #include <assert.h>
 
+#include <inttypes.h>
 #include "dvdreadsrc.h"
 
 #include <dvdread/dvd_reader.h>
